src/components/NewActivityList.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react'
+import { Table, Button, Modal, ControlLabel, FormControl } from 'react-bootstrap'
+import './components.css'
+import firebase from 'firebase'
+import toastr from 'toastr'
+import moment from 'moment'
+import CountDown from './CountDown'
+
+class NewActivityList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hours: '',
+      money: '',
+      how_long: true,
+      startedAt: '',
+      isModalShown: true
+    }
+  }
+
+  componentDidMount() {
+    const userId = firebase.auth().currentUser.uid;
+    firebase.database().ref('/users/' + userId + '/settings').once('value').then(snapshot => {
+      const currentRate = (snapshot.val().currentRate * 0.4)
```

Comment:
how about fetching user and saving to state in App.js and then passing to Child routes?